error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StringRules.java

```diff
@@ -162,6 +162,22 @@ String after(Object object) {
     }
   }
 
+  /**
+   * Prefer direct invocation of {@link String#copyValueOf(char[])} over the indirection introduced
+   * by {@link String#copyValueOf(char[])}.
+   */
+  static final class StringCopyValueOf {
```

Comment:
We generally name the rule after the `@AfterTemplate` code. I this case it's a bit tricky, but I'll propose something.